src/sentry/api/endpoints/organization_artifactbundle_assemble.py

```diff
@@ -0,0 +1,79 @@
+import jsonschema
+from rest_framework.request import Request
+from rest_framework.response import Response
+
+from sentry.api.base import region_silo_endpoint
+from sentry.api.bases.organization import OrganizationReleasesBaseEndpoint
+from sentry.api.exceptions import ResourceDoesNotExist
+from sentry.tasks.assemble import (
+    AssembleTask,
+    ChunkFileState,
+    get_assemble_status,
+    set_assemble_status,
+)
+from sentry.utils import json
+
+
+@region_silo_endpoint
+class OrganizationArtifactBundleAssembleEndpoint(OrganizationReleasesBaseEndpoint):
+    def post(self, request: Request, organization) -> Response:
+        """
+        Assembles an artifact bundle and stores the debug ids in the database.
+        """
+        if not self.has_release_permission(request, organization, None):
+            raise ResourceDoesNotExist
+
+        schema = {
+            "type": "object",
+            "properties": {
+                "projects": {"type": "array", "items": {"type": "integer"}},
```

Comment:
I actually do not know the type of the project on the client. This should be a list of slugs instead.

Reply:
Is there any way to convert the project slug to id without making queries? Just asking because it we could already supply ids it would make everything simpler.